### ISSUE TITLE
xsecurelock does appear to be grabbing focus

### ISSUE THREAD

**Issue by dan9550** [google/xsecurelock#50]
I just compiled and install xsecurelock i have xscreensaver installed also.

If i run "env XSECURELOCK_NO_COMPOSITE=1 XSECURELOCK_BURNIN_MITIGATION=200 XSECURELOCK_SAVER=saver_xscreensaver xsecurelock" the screen with turn blank and have a password prompt but i cannot interact with xsecure lock. Also pressing the super key will drop me back to gnome-shell activity view, i can press control-c to quit the xsecurelock process. It seems like the terminal still has focus this entire time.

I am running Fedora 28 (4.18.10-200.fc28.x86_64) using Gnome Shell on Wayland. Surely i'm just doing something wrong here?

Small update: In my last test if i change the application focus in the activities overview to say Chromium or VS Code my inputs are sent to xsecurelock, having focus to gnome-terminal or nautilus xsecurelock is left in the dark. I have tried using "XSECURELOCK_FORCE_GRAB=2".

My last test:
`env XSECURELOCK_NO_COMPOSITE=1 XSECURELOCK_BURNIN_MITIGATION=200 XSECURELOCK_SAVER=saver_xscreensaver xsecurelock
2018-10-12T09:42:54Z 27477 xsecurelock: XComposite extension detected but disabled by user.
2018-10-12T09:42:54Z 27478 xsecurelock: Executable 'saver_xscreensaver' must be executable.`

**Comment by soumya92** [comment]
> using Gnome Shell on Wayland

Wayland is completely different from X11, and xsecurelock will not work on it.
The reason it works in VSCode/Chromium is probably because they haven't been updated for wayland yet, so they're running under XWayland or something similar, within which xsecurelock will mostly work.

**Comment by divVerent** [comment]
Yes, it just can't work on Wayland. Use whatever lock your Wayland server supports. It is usually part of the server process, so it should be the most secure you can get - if your lock crashes, it will take your session with it.

**Comment by dan9550** [comment]
Sorry guys. I must have miss read. Alas I continue searching for a way to use xscreensaver and lock in Wayland.

**Comment by divVerent** [comment]
It seems rather unlikely that most of xscreensaver's "hacks" (which is how
jwz calls the animations) will ever be ported to Wayland, as they heavily
rely on Xlib APIs.

Most likely Wayland will require a start from scratch regarding the
animations.
